packages/cashier-v2/tailwind.config.ts

```diff
@@ -0,0 +1,10 @@
+import type { Config } from 'tailwindcss';
+import QuillTailwindConfig from '@deriv/quill-design/quill-tailwind/tailwind.config.cjs';
+
+export default {
+    content: ['./src/**/*.{js,jsx,ts,tsx}'],
+    corePlugins: {
+        preflight: false,
+    },
```

Comment:
Here as well you need to add:
```suggestion
         important: "#cashier_v2"
    },
```